Source/Csla.AspNetCore/Csla.AspNetCore.csproj

```diff
@@ -28,10 +28,12 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Authorization" Version="5.0.11" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Authorization" Version="5.0.11" />
```

Comment:
Presumably this would ideally be 6.0.1, but that would require an update to all of the related packages for net6.0 throughout, which you might be delaying on purpose?

Reply:
Yeah, there's a need to go through and check all the versions. dependabot is nice, but falls down when your projects target multiple versions of dotnet that need different versions of dependencies.